3.distributed-calculator/react-calculator/server.js

```diff
@@ -8,7 +8,8 @@ const port = 8080;
 const daprPort = process.env.DAPR_HTTP_PORT || 3500;
 
 const daprUrl = `http://localhost:${daprPort}/v1.0/invoke`;
-const stateUrl = `http://localhost:${daprPort}/v1.0/state`;
+const stateStoreName = `statestore`;
```

Comment:
It would be better to use something a bit more specific here, so it's clear to the user that this is *user-defined*.  Maybe `mystatestoreabc123`.

Reply:
The cli generates the default name as statestore. a comment here to indicate that its from the components yaml would suffice here.

Reply:
Decided on using a link to the line where we set it, added the link.